webapp/src/store/actions/courses.js

```diff
@@ -0,0 +1,16 @@
+import {createAction} from 'redux-api-middleware';
+import {ENDPOINT} from '../../config';
+
+export const GET_COURSES_REQUEST = 'GET_COURSES_REQUEST';
+export const GET_COURSES_SUCCESS = 'GET_COURSES_SUCCESS';
+export const GET_COURSES_ERROR = 'GET_COURSES_ERROR';
+
+export const getCourses = () => createAction({
+  endpoint: `${ENDPOINT}/api/courses/?limit=11000`,
```

Comment:
I wonder if we can come up with a way to get around this hardcoded limit... I've been trying to come up with solutions for a while but the only things I can think of are slow or have a host of other issues, for example, hitting the API once to see how many results there are, then making the real request to get all of them. We should chat at some point and try to brainstorm ideas

Reply:
We could just replace 11,000 with an absurdly large number. 😛 
Otherwise we could add a method to the API that returns all results without pagination.

More generally, I was thinking that if we modified the backed to just be a script that spewed out a 2 MB JSON file, the site could be completely static - but that limits us in the future from adding features that actually require a backend store.

Reply:
We could just disable pagination on the API end, but that would make the browsable API (currently able to be exposed in planned "prod") basically unusable in browsers, which sort of defeats the point of making it available. I created issue #25 for this since it's not really needed for the scope of this PR

Reply:
The easiest and least "guessy" solution we could have here is just to create a new `meta` endpoint in the api that contains some basic information about the data set.  Perhaps some info such as `last_updated`, `num_transfers`, etc.

That'll probably be something with it's own PR and feature improvement if we were to go that route, however.

Reply:
Moving to #25.